src/pages/Sessions.tsx

```diff
@@ -65,9 +66,12 @@ const Sessions = () => {
   ] = useState<FamilyDetailResponse | null>(null);
   const [snackbarMessage, setSnackbarMessage] = useState("");
 
+  const [isLoadingSession, setIsLoadingSession] = useState(true);
```

Comment:
maybe this should have a default value of false, since we are setting it to true in the useEffect hooks

Reply:
hmm yeah since it gets set true every time i guess this value doesn't matter much. i think it's nice to keep it as true because it communicates that things will be loading on mount 😄 lmk what you think